lib/migration_view.rb

```diff
@@ -132,12 +132,14 @@ def self.drop_view(view, cascade = true)
     File.delete("db/views/#{view}.sql") if File.exist?("db/views/#{view}.sql")
 
     Rails.logger.info("MigrationView::drop_view: drop the view: #{view}")
-    sql = "DROP VIEW IF EXISTS #{view}"
-    if cascade
-      sql = sql + " CASCADE"
+    if MigrationView::view_exists?(view)
+     sql = "DROP VIEW IF EXISTS #{view}"
+      if cascade
+        sql = sql + " CASCADE"
+      end
+      # Rails.logger.debug("MigrationView::drop_view: sql: #{sql}")
```

Comment:
I don't see that this is any different.

What was your intended goal?

Reply:
to check to see if the view exists before executing drop.